concepts/identity.md

```diff
@@ -0,0 +1,283 @@
+# META Identity
+
+This document outlines the concept of a META Identity. A META Identity is used
+to both identify participants in the network using a cryptographic identifier
+and to provide verifiable information about those participants.
+
+## Status of this Document
+
+The META protocol is at an early stage of development, expect this document to
+change frequently during this early phase.
+
+## Outline
+
+A META Identity is a representation of a participant in the META network. It is
+designed to be a self-sovereign identity with two functions:
+
+- Referable unique identifier for a participant within the META network
+
+- Enable web of trust mechanisms through inclusion and signing of external
+  identifiers relating to the participant (eg. META-ID = [Spotify-ID, IPI,
+  National ID, PGP key, Twitter handle])
+
+## Terminology
+
+*TODO*: complete definitions of terms used in this document
+
+- Custodian
+- Smart Contract
+- Identity Owner
+- META-ID
+- Identity Signature Scheme
+- Identity Custody Scheme
+- Trust Item - a detached cryptographic signature of a metadata item
+
+sha3() indicates keccak256 hashing of the contents within parentheses.
+
+## Core concepts
+
+A Meta Identity MUST define:
+
+- META-ID: A 32 byte ENS namehash
+- CID: A content-addressed identifier
+
+The META-ID is the UNIQUE identifier of the Meta Identity. It is IMMUTABLE. There are no constraints on the hash data. The META-ID is ONLY an index key, and MUST NOT be used for cryptographic purposes.
+
+The CID MUST point to a valid META Object. The META Object MUST include a property "meta." The value of this property MUST be a data structure as defined in subsection "Core" under "Metadata."
+
+The manifest CAN include an arbitrary number of metadata items. These define claims of auxiliary identifiers linked to the Meta Identity. These SHOULD define data structures as defined in subsection "Auxiliary" under "Metadata."
+
+### Ownership and permissions
+
+A Meta Identity MUST be owned by one single Ethereum address. The mapping of address to Meta Identity MUST be recorded on the blockchain. Ownership of the Meta Identity MUST be enforced through the Identity Custody Scheme. The initial Meta Identity owner SHOULD be the transaction sender of the Meta Identity instantiation.
```

Comment:
We should standardise on either `Meta` or `META`, I vote for `META` which is inline with the rest of the concepts documents.

Reply:
We should either say "a public blockchain" or "the Ethereum blockchain" as it isn't clear what "the blockchain" refers to.